### PR TITLE
get specifc book

### PR DESCRIPTION
**Pull request by MohammadAlHabil**
relates #64

### CODE REVIEW

File: server/controllers/book/getUserBooks.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { RequestHandler } from 'express';
+import { getUserBooksQuery } from '../../queries';
+
+const getUserBooks: RequestHandler = async (req, res, next) => {
+  try {
+    const { id } = req.params;
```
Comment (Mu7ammadAbed) [inline server/controllers/book/getUserBooks.ts:6]: validate this

File: server/controllers/book/getBook.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import { RequestHandler } from 'express';
+import { getBookQuery } from '../../queries';
+
+const getBook: RequestHandler = async (req, res, next) => {
+  try {
+    const { id } = req.params;
```
Comment (Mu7ammadAbed) [inline server/controllers/book/getBook.ts:6]: validate

File: server/controllers/book/getBook.ts [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+    const { rows } = await getBookQuery(+id);
+    res.json({ message: 'Successfully retrieved all books', status: 200, data: rows });
```
Comment (Mu7ammadAbed) [inline server/controllers/book/getBook.ts:8]:
```suggestion
    res.json({ message: 'Successfully retrieved appointment data', status: 200, data: rows });
```

File: server/controllers/book/getUserBooks.ts [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+    const { rows } = await getUserBooksQuery(+id);
+    res.json({ message: 'Successfully retrieved all books', status: 200, data: rows });
```
Comment (Mu7ammadAbed) [inline server/controllers/book/getUserBooks.ts:8]:
```suggestion
    res.json({ message: 'Successfully retrieved all user bookings', status: 200, data: rows });
```

**Review by Mu7ammadAbed — APPROVED** [review]

**Review by Jehad91 — APPROVED** [review]

**Review by MostafaQanoo — APPROVED** [review]

**Review by IsraaHamdi — APPROVED** [review]

**Pull request merged**